packages/mikado_reborn/src/components/NavItem/NavItem.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <li :class="classes">
     <a :href="href" @click="emitClick">
-      <mkr-icon v-if="icon" :name="icon"/>
+      <slot name="icon" />
+      <mkr-icon v-if="icon && !$scopedSlots.icon" :name="icon"/>
```

Comment:
I think it should work that way too
```suggestion
      <slot name="icon">
            <mkr-icon v-if="icon" :name="icon" />
      </slot>
```

Reply:
Yes that's even better thx ! 🙏